framework/include/wifi_manager/wifi_manager.h

```diff
@@ -0,0 +1,129 @@
+/****************************************************************************
+ *
+ * Copyright 2016 Samsung Electronics All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND,
+ * either express or implied. See the License for the specific
+ * language governing permissions and limitations under the License.
+ *
+ ****************************************************************************/
+
+#ifndef WIFI_MANAGER_H
+#define WIFI_MANAGER_H
+
+typedef enum {
+	// STA mode status
+	AP_DISCONNECTED,
+	AP_CONNECTED,
+
+	// SOFT AP mode status
+	CLIENT_CONNECTED,
+	CLIENT_DISCONNECTED
+} connect_status;
+
+/**
+ * @brief <b> wifi result type FAIL, SUCCESS, INVALID ARGS</b>
+ */ 
+typedef enum {
+	WIFI_MANAGER_FAIL = -1,
+	WIFI_MANAGER_SUCCESS,
+	WIFI_MANAGER_INVALID_ARGS,
+	WIFI_MANAGER_TIMEOUT,
+	WIFI_MANAGER_BUSY,
+} wifi_manager_result;
+
+
+typedef enum {
+	STA_MODE,
+	SOFT_AP_MODE
+} wifi_manager_mode;
+
+
+typedef struct {
+    void (*sta_connected) (void);
+    void (*sta_disconnected) (void);
+    void (*softap_sta_join) (void);
+    void (*softap_sta_leave) (void);
+} wifi_manager_cb;
+
+
+typedef struct {
+	char ip4_address[18];
+	char ssid[32];
+    int rssi;
```

Comment:
Here are violations of coding rule. Please use tab as an indention.

Reply:
I did apply fomatter.sh